### PR TITLE
docs: use consistent uppercase "J"

### PR DESCRIPTION
**Pull request by CodingSpiderFox**
---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [x] Tests are added where necessary
- [x] The JDL part is updated if necessary
- [x] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [x] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (below reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

**Review by DanielFran — APPROVED** [review]

**Comment by CodingSpiderFox** [comment]
The PR says

"Required statuses must pass before merging
All required [statuses](https://docs.github.com/rest/commits/statuses) and check runs on this pull request must run successfully to enable automatic merging.
Auto-merge enabled

This pull request will merge automatically when all requirements are met."

Do I need to do something? Trigger the CI somehow?

**Pull request merged**
